src/WebJobs.Extensions.DurableTask/DurableOrchestrationContext.cs

```diff
@@ -260,11 +309,11 @@ public Task CallFunctionWithRetryAsync(string functionName, RetryOptions retryOp
         /// </exception>
         public async Task<TResult> CallFunctionAsync<TResult>(string functionName, params object[] parameters)
```

Comment:
Would it make sense to name this `CallActivityAsync` or `CallActivityFunctionAsync` to be symmetrical? I realize this would be a breaking change but I wonder if it's confusing to have a `CallFunctionAsync` method that works with activity functions but not orchestrator functions.

Reply:
Yes, I was leaning towards that but worried about breaking change as well.  I like CallActivityAsync.

---
In reply to: [138217536](https://github.com/Azure/azure-functions-durable-extension/pull/57#discussion_r138217536) [](ancestors = 138217536)